### PR TITLE
Bug fixes

### LINKED ISSUE #74: Border between Minsk (UMMV) and Baltic region (EYVL & EVRR) duplicate border
There is 2 different borders between Minsk FIR (UMMV) and Vilnius FIR (EYVL) & Riga FIR (EVRR).

image

### LINKED ISSUE #80: MHCC_CTR not showing on map
I have tried investigating this but without finding an answer for it.
MHCC is not showing on VATSpy, but seems to cover either the whole MHTG or somewhere in MHTG's FIR, however on skyvector MHTG FIR does not excist, it only shows MHCC in that area. 

Looking on vatsim stats, both MHTG_CTR and MHCC_CTR seems to be actively in use. 

Controller info of MHCC_CTR were: "Cenamer Control | English & Spanish^Â§vatca.net" and had ATIS setup at MHTG airport.

### PR DESCRIPTION
**Pull request by Adrian2k**
Resolves #80, resolves #74

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -17863,7 +17863,7 @@ LWSS|Skopje||LWSS
 LYBA|Beograd||LYBA
 LZBB|Bratislava||LZBB
 MDCS|Santo Domingo||MDCS
-MHTG|Central America||MHTG
```
Comment (nepetadosmil) [inline VATSpy.dat:17866]: Just clarifying something about the position name with them; should be fine to merge after

Comment (nepetadosmil) [reply VATSpy.dat:17866]: New position's (MHCC_CTR) radio callsign is "Cenamer Centre"

**Comment by Adrian2k** [comment]
I think I have renamed it to Cenamer Centre now, but yet to see it online to see it is working. The FIR is called Central America.

**Comment by Adrian2k** [comment]
I have renamed Central America FIR to Cenamer FIR now and added callsign Centre. However the FIR is actually called Central America.

Ready for merge now.

**Pull request merged**